bin/gaffer

```diff
@@ -268,6 +268,30 @@ if [[ -n $DELIGHT ]] ; then
 
 fi
 
+# Set up 3rd Party extensions
+##########################################################################
+
+for extension in `echo $GAFFER_EXTENSION_PATHS | sed 's/:/\n/g'`; do
```

Comment:
This appears to be thwarted by spaces - a problem we've had elsewhere in the wrapper before :

```
GAFFER_EXTENSION_PATHS="/tmp/something with space/:/tmp/somethingElse" gaffer
Loading extension: /tmp/something
Loading extension: with
Loading extension: space/
Loading extension: /tmp/somethingElse
```

[This commit](https://github.com/johnhaddon/gaffer/commit/dc7ed418e4426b2e0cfc3e5b0b0350344d90bb24) seems to be one approach that might work...